### ISSUE TITLE
What should I do if I want to close the pretrain process?

### ISSUE THREAD

**Issue by consistent1997** [leoxiaobin/deep-high-resolution-net.pytorch#120]
Dear leoxiaobin, I'm a second grade postgraduate. I want to improve something, such as the quantity of parameters, accuracy and so on, on a basis of your HRNet. Since the pretrain process on ImageNet is considerably huge and our configuration is too poor to pretrain this network, so I want to know how to modify the code to close the pretrain process. Can I modify "is_Train=True" into "is_Train=False" or modify "_C.MODEL.INIT_WEIGHTS = True" into "_C.MODEL.INIT_WEIGHTS = False"? Thank you very much if you can give some suggestions!

**Comment by leoxiaobin** [comment]
You can set _C.MODEL.PRETRAINED = '' for that.

**Comment by yuechengyu0710** [comment]
> Dear leoxiaobin, I'm a second grade postgraduate. I want to improve something, such as the quantity of parameters, accuracy and so on, on a basis of your HRNet. Since the pretrain process on ImageNet is considerably huge and our configuration is too poor to pretrain this network, so I want to know how to modify the code to close the pretrain process. Can I modify "is_Train=True" into "is_Train=False" or modify "_C.MODEL.INIT_WEIGHTS = True" into "_C.MODEL.INIT_WEIGHTS = False"? Thank you very much if you can give some suggestions!

Hello, I'm also a second grade postgraduate. I also have some ideas that I want to modify on HRNet. Can we exchange them?this is my e-mail yuechengyu0710@live.com .looking forward to your reply

**Comment by consistent1997** [comment]
Thank you very much！I get it.
On 10/11/2019 17:10，Leo Xiao<notifications@github.com> wrote：

You can set _C.MODEL.PRETRAINED = '' for that.
